globaltaggingv1/global_tagging_v1_examples_test.go

```diff
@@ -190,13 +186,8 @@ var _ = Describe(`GlobalTaggingV1 Examples Tests`, func() {
 			fmt.Println("\nDetachTag() result:")
 			// begin-detach_tag
 
-			resourceModel := &globaltaggingv1.Resource{
-				ResourceID: &resourceCRN,
-			}
 
-			detachTagOptions := globalTaggingService.NewDetachTagOptions(
-				[]globaltaggingv1.Resource{*resourceModel},
-			)
+			detachTagOptions := globalTaggingService.NewDetachTagOptions()
```

Comment:
Please remove 1 blank line from here. Currently there are 2 here, but it should be only 1.